functors_task_1/student/include/impl.h

```diff
@@ -16,16 +30,18 @@ struct BeerOrganizer
  *
  * @note Only Corona and HoeGaarden are expensive
  */
-bool isExpensiveBeer(/**???*/)
+bool isExpensiveBeer(BeerBrand beer)
 {
+    return beer == BeerBrand::Corona || beer == BeerBrand::HoeGaarden ? true : false;
 }
 
 /**
  * @todo Implement lambda beer country equality comparator
  * @return true if beer county is the same, false otherwise
  */
-auto sameCountry = [](/**???*/)
+auto sameCountry = [](BeerBrand beer1, BeerBrand beer2)
 {
+    return (int)getBeerCountry(beer1) == (int)getBeerCountry(beer2);
```

Comment:
why do you need this cast?

Reply:
I don't remember clearly, but I think it didn't work without the cast

Reply:
try to remove it - this should be safe